libs/dh/market-participant/actors/feature-actors/src/lib/dh-actors-overview.component.ts

```diff
@@ -38,15 +40,42 @@ interface Actor {
       }
     `,
   ],
-  imports: [TranslocoModule, WATT_TABLE, WATT_CARD],
+  imports: [TranslocoModule, DhActorsFiltersComponent, WATT_TABLE, WATT_CARD],
 })
-export class DhActorsOverviewComponent {
+export class DhActorsOverviewComponent implements OnInit, OnDestroy {
+  private apollo = inject(Apollo);
+  private destroy$ = new Subject<void>();
+
+  getActorsQuery$ = this.apollo.watchQuery({
+    useInitialLoading: true,
+    notifyOnNetworkStatusChange: true,
+    query: GetActorsDocument,
+  });
+
   dataSource = new WattTableDataSource<Actor>([]);
 
   columns: WattTableColumnDef<Actor> = {
-    glnOrEic: { accessor: 'glnOrEic' },
+    glnOrEicNumber: { accessor: 'glnOrEicNumber' },
     name: { accessor: 'name' },
     marketRole: { accessor: 'marketRole' },
     status: { accessor: 'status' },
   };
+
+  filters$ = new BehaviorSubject<ActorsFilters>({
+    actorStatus: null,
+    marketRoles: null,
+  });
+
+  ngOnInit(): void {
+    this.getActorsQuery$.valueChanges.pipe(takeUntil(this.destroy$)).subscribe({
```

Comment:
Suggestion: Store the subscription reference of valueChanges, and call unsubscribe on that in ngOnDestory